push/management/commands/push_notifications.py

```diff
@@ -21,7 +21,12 @@ def on_notification(body, message: kombu.message.Message):
             len(message.body),
         )
         try:
-            notification = Notification(**body)
```

Comment:
I propose to leave this as is. But in `Notification.to_dict()` replace `extra=self.extra` by `**self.extra`